templates/docs/examples/patterns/equal-height-row/4-item-columns.html

```diff
@@ -0,0 +1,127 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Equal height row / 4 item columns{% endblock %}
+
+{% block standalone_css %}patterns_equal-height-row{% endblock %}
+
+{% block content %}
+
+<div class="row--25-75-on-large">
+  <div class="col">
+    <a href="#">
+      <p class="p-muted-heading">Latest from our blog</p>
+    </a>
+  </div>
+  <div class="col">
+    <div class="p-equal-height-row has-divider-below-item-1 has-divider-below-item-3">
+      <div class="p-equal-height-col">
```

Comment:
`p-equal-height-row__col`

Although I'm wondering if it would make sense to "abstract" it away from row/col a bit more and do: `p-equal-height-items` `p-equal-height-items__item`? Hmm… that doesn't look better actually…